framework/src/main/java/org/checkerframework/common/reflection/DefaultReflectionResolver.java

```diff
@@ -487,10 +488,20 @@ private boolean isUnknownMethod(MethodInvocationTree tree) {
         Names names = Names.instance(context);
 
         List<Symbol> result = new ArrayList<>();
+        Method loadClass = null;
+        Symbol sym = null;
         try {
-            Method loadClass = Resolve.class.getDeclaredMethod("loadClass", Env.class, Name.class);
+            loadClass = Resolve.class.getDeclaredMethod("loadClass", Env.class, Name.class);
             loadClass.setAccessible(true);
-            Symbol sym = (Symbol) loadClass.invoke(resolve, env, names.fromString(className));
+            sym = (Symbol) loadClass.invoke(resolve, env, names.fromString(className));
```

Comment:
Shouldn't these two calls be split up?
One depends on the javac API and should always work.
The other is reflectively looking for user code and might legitimately fail and only give a warning.